.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,9 +26,16 @@ DO NOT REMOVE THE FOLLOWING CHANGELOG LINES, EVEN IF YOU DON'T USE THEM.-->
 ## Explanation for Changes
 <!-- How do your changes improve the library?
 
-For PRs introducing new features, please provide code snippets using the
-newly introduced functionality and ideally even the expected rendered output.
+For PRs introducing new features, please adjust the link below to reference the docs of your feature.
+These should provide code snippets using the newly introduced functionality and ideally
+even the expected rendered output.
 -->
+[Documentation Reference](https://manimce--####.org.readthedocs.build/en/####/reference/path.to.page.html) 
+<!-- In the link above replace #### with you PR number and adjust the path to the module / class
+you worked on. This could look like /manim.mobject.geometry.Arc.html for the Arc class.
+Notice that the link will only work once the documentation is build which may take
+a few minutes.-->
+
```

Comment:
LGTM, but in gerneral I would add somewhere that we don't expect people to add this.
I would formulate it something more like:
"The community devs encourage you to provide a link to the docs ...  for quick access"
There are already so many steps one has to do to make a pull request, and I am concerned that new contributors would feel a bit overwhelmed, and might decide to not make a pr.